docs/docs/loading-data-asynchronously.md

```diff
@@ -0,0 +1,114 @@
+---
+id: loading-data-asynchronously
+title: Loading Data Asynchronously
+permalink: docs/loading-data-asynchronously.html
+prev: typechecking-with-proptypes.html
+---
+
+
+Often, the data that a component needs is not available at initial render. We can load data asynchronously in the `componentDidMount` [lifecycle hook](https://facebook.github.io/react/docs/react-component.html#componentdidmount).
+
+In the following example we use the `fetch` API to retrieve information about Facebook's Gists on GitHub and store them in the state.
```

Comment:
We should mention that this is a browser API but it requires a polyfill. Link to the polyfill. Mention that if you use Create React App, polyfill is already included.